tagging/github_workflow_commands.py

```diff
@@ -0,0 +1,41 @@
+"""
+GitHub Workflow Commands (gwc) for GitHub Actions can help us pass information
+from a Workflow's Job's various build steps to others via "output" and improve
+the presented logs when viewed via the GitHub web based UI.
+
+Reference: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions
+
+Workflow commands relies on emitting messages:
+
+    print("::{command name} parameter1={data},parameter2={data}::{command value}")
+
+The functions defined in this file will only emit such messages if found to be
+in a GitHub CI environment.
+"""
+
+import json
+import os
+
+
+def _gwc(command_name, command_value="", **params):
```

Comment:
@consideRatio is this function used at all?
I think I can safely delete it as well?

Reply:
Yepp, and the `import json` statement then.

Reply:
I also renamed the file and removed the docs, because they seem to be irrelevant